### PR TITLE
#24 - Remove :refer :all usage in human-generated files

### PR DESCRIPTION
**Pull request by pfeodrippe**
Fixes https://github.com/naomijub/gXTDB/issues/24.

### CODE REVIEW

File: src/gxtdb/server.clj [hunk 1]
```diff
@@ -35,6 +35,16 @@
...
 
+(comment
+  ;; For the REPL.
+
+  (def *server (atom nil))
+
+  (do (some-> @*server server/stop)
+      (reset! *server (-main)))
+
+  ())
```
Comment (naomijub) [inline src/gxtdb/server.clj:46]: Do you need this line?

Comment (naomijub) [reply src/gxtdb/server.clj:46]: `()`

**Comment by naomijub** [comment]
LFTM, do you think that now lein eastwood can be added to the pipeline?
I think it was complaining about the test section. Would just need to ignore the auto generated files

**Comment by pfeodrippe** [comment]
> LFTM, do you think that now lein eastwood can be added to the pipeline? I think it was complaining about the test section. Would just need to ignore the auto generated files

Sure sure, I can take care of that o/ Where do you see the complaining happening?

File: src/gxtdb/server.clj (hunk 1 above)
Comment (pfeodrippe) [reply src/gxtdb/server.clj:46]: I always put this as I find that's easy to mistake the `)` from the `comment` with the last expression. But I can remove it if you want o/

Comment (pfeodrippe) [reply src/gxtdb/server.clj:46]: *Mistaking when you are evaluating

**Comment by pfeodrippe** [comment]
> > LFTM, do you think that now lein eastwood can be added to the pipeline? I think it was complaining about the test section. Would just need to ignore the auto generated files
> 
> Sure sure, I can take care of that o/ Where do you see the complaint happening?

Oh ok, you are saying to enable eastwood, do you prefer that to clj-kondo? Or maybe you see that they are doing different things?

**Comment by naomijub** [comment]
> Oh ok, you are saying to enable eastwood, do you prefer that to clj-kondo? Or maybe you see that they are doing different things?

I have not used clj-kondo in the CI, might be a nice try. Somehow I use both of them and don't see them as conflicting, but only one of them should be good.

**Comment by pfeodrippe** [comment]
> > Oh ok, you are saying to enable eastwood, do you prefer that to clj-kondo? Or maybe you see that they are doing different things?
> 
> I have not used clj-kondo in the CI, might be a nice try. Somehow I use both of them and don't see them as conflicting, but only one of them should be good.

Got it, do we want to merge this one and I can create a PR adding both Clj-kondo and Eastwood linters with the necessary fixes?

**Comment by naomijub** [comment]
Maybe this Pr? Already addresses most of it anyway

**Comment by pfeodrippe** [comment]
> Maybe this Pr? Already addresses most of it anyway

I am working on the entity feature on top of this PR, so wanted to avoid adding another WIP on top of this PR, I've created an issue already for the CI linters, wdyt?  

So we can maintain our 70/30 feature/tech debt allocated time lol uaehuheuaehuuae

I would prefer to go with the entity, if possible, but you tell me \o

**Review by evaporei — APPROVED** [review]

**Comment by pfeodrippe** [comment]
Thank you, @evaporei and @naomijub \o/

**Pull request merged**